googlemock/src/gmock.cc

```diff
@@ -48,6 +48,13 @@ GMOCK_DEFINE_string_(verbose, internal::kWarningVerbosity,
                      "  warning - prints warnings and errors.\n"
                      "  error   - prints errors only.");
 
+GMOCK_DEFINE_int32_(default_mock_behavior, 1,
```

Comment:
What happens if someone sets this to an invalid value?
Unless we prevent it, the static_cast above can have undefined behavior.